app/controllers/api/v1/tasks_controller.rb

```diff
@@ -28,13 +29,25 @@ def create
                         error: "Please fill out all required fields"
                     }
                 end
+
+                user = User.new(params[:user])
+
+                if user.valid>
+                    user.save
+                    render json: user
+                else
+                    render json: {
+                        error: "User not saved"
+                    }
+                end
             end
 
             private
 
-            def task_params
-                params.require(:content, :parent_id, :skill_id).permit(:volunteer_id, :task_language, :english_proficiency, :childs_age)
-            end
+            # def task_params
```

Comment:
I think we should still use strong_params (like the bit commented out) just because this is user info so even more risky - there is a way to nest them, let me just have a quick look